src-test/org/etools/j1939_84/model/OBDModuleInformationTest.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2020. Equipment & Tool Institute
+ */
+package org.etools.j1939_84.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.etools.j1939_84.bus.j1939.packets.SupportedSPN;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Unit Test for the {@link OBDModuleInformation} class
+ *
+ * @author Marianne Schaefer (marianne.m.schaefer@gmail.com)
+ *
+ */
+public class OBDModuleInformationTest {
+
+    private static List<SupportedSPN> makeListOfSupportedSPNs(int[] data) {
+        List<SupportedSPN> supportedSpnsList = new ArrayList<>();
+        SupportedSPN supportedSpn = new SupportedSPN(data);
+        supportedSpnsList.add(supportedSpn);
+        return supportedSpnsList;
+    }
+
+    private OBDModuleInformation instance;
+
+    private OBDModuleInformation instance2;
+
+    @Before
+    public void setUp() throws Exception {
+
+        instance = new OBDModuleInformation(0);
+        instance.setObdCompliance((byte) 4);
+        instance.setSupportedSpns(makeListOfSupportedSPNs(new int[] { 4, 5, 6, 7, 8 }));
+
+        instance2 = new OBDModuleInformation(0);
+        instance2.setObdCompliance((byte) 4);
+        instance2.setSupportedSpns(makeListOfSupportedSPNs(new int[] { 4, 5, 6, 7, 8 }));
+
+    }
+
+    @Test
+    public void testEquals() {
+        assertFalse(instance.equals(new Object()));
+        assertFalse(instance.equals(null));
+        assertTrue(instance.equals(instance));
+        assertTrue(instance.equals(instance2));
+    }
+
+    @Test
+    public void testGetDataStreamSpns() {
+        assertTrue(instance.getDataStreamSpns().equals(instance2.getDataStreamSpns()));
+    }
+
+    @Test
+    public void testGetFreezeFrameSpns() {
+        assertTrue(instance.getFreezeFrameSpns().equals(instance2.getFreezeFrameSpns()));
```

Comment:
This needs to test that only the FreezeFrame Spns are return, not that they are the same.  This applies for the GetDataStreams as well.